axes/apps.py

```diff
@@ -1,16 +1,36 @@
+from __future__ import absolute_import
+
+import logging
+import warnings
+
 from django import apps
 
+from .conf import settings
+
+
+log = logging.getLogger(settings.AXES_LOGGER)
+
+
+def patch_login_view():
+    warnings.warn(
+        'Automatic patching of LoginView using AXES_USE_LEGACY_LOGIN_VIEW_PATCH is deprecated. '
+        'use axes.views.AxesLoginView instead, or add axes.views.AxesLoginMixin to your custom login view.',
+        DeprecationWarning
+    )
+    from django.contrib.auth.views import LoginView
```

Comment:
These could be imported on the top of the module, no need to separate the imports module and function level here imo.